app/views/vacancies/_job_details_section.html.haml

```diff
@@ -0,0 +1,68 @@
+%section#job-details.govuk-tabs__panel.no-border-override
+  %h2.govuk-heading-l 
+    = t('jobs.job_details')
+  %table.govuk-table
+    %tbody.govuk-table__body
+      %tr.govuk-table__row
+        %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" } 
+          = t('jobs.subject')
+        %td.govuk-table__cell
+          - if @vacancy.other_subjects.nil? || @vacancy.other_subjects == ''
+            = @vacancy.main_subject
+          - else 
+            = @vacancy.main_subject + ', ' + @vacancy.other_subjects
+      %tr.govuk-table__row
+        %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" } 
+          = t('jobs.working_patterns')
+        %td.govuk-table__cell
+          = @vacancy.working_patterns
+          -# - if @vacancy.flexible_working?
```

Comment:
Unless you're putting it in as an example for a TODO, like you do below, just take commented out code out, please.